### PR TITLE
Handle exceptions for int input range

### PR DESCRIPTION
**Pull request by preetha-intel**
Bug fix in mapping between V1 and V2 provider options

### CODE REVIEW

**Comment by sfatimar** [comment]
This seems ok. Does this need to be merged

**Comment by sfatimar** [comment]
LGTM

File: onnxruntime/core/providers/openvino/openvino_provider_factory.cc [hunk 1]
```diff
@@ -105,13 +105,25 @@ struct OpenVINO_Provider : Provider {
...
+        num_streams = 1;
+        LOGS_DEFAULT(WARNING) << "[OpenVINO-EP] The value for the key 'num_streams' should be in the range of 1-8.\n "
+                         << "Executing with num_streams=1";
+        // ORT_THROW("[ERROR] [OpenVINO] The value for the key 'num_streams' should be in the range of 1-8 \n");
+      }
+      if (num_streams>8) {
+        num_streams = 8;
+        LOGS_DEFAULT(WARNING) << "[OpenVINO-EP] The value for the key 'num_streams' should be in the range of 1-8.\n "
+                         << "Executing with maximum num_streams=8.";
+        // ORT_THROW("[ERROR] [OpenVINO] The value for the key 'num_streams' should be in the range of 1-8 \n");
```
Comment (sfatimar) [inline onnxruntime/core/providers/openvino/openvino_provider_factory.cc:126]: remove the commented code

File: onnxruntime/core/providers/openvino/openvino_provider_factory.cc [hunk 2]
```diff
@@ -105,13 +105,25 @@ struct OpenVINO_Provider : Provider {
       num_of_threads = std::stoi(provider_options_map.at("num_of_threads"));
       if (num_of_threads <= 0) {
         num_of_threads = 1;
+        LOGS_DEFAULT(WARNING) << "[OpenVINO-EP] The value for the key 'num_threads' should be in the positive range.\n "
+                         << "Executing with num_threads=1";
+        // ORT_THROW("[ERROR] [OpenVINO] The value for the key 'num_threads' should be in the positive range\n");
```
Comment (sfatimar) [inline onnxruntime/core/providers/openvino/openvino_provider_factory.cc:110]: remove the commented code

**Comment by preetha-intel** [comment]
Added part of 23.1 release. Closing as this is redundant.

**Pull request closed without merging**